gnucash/gnome/window-autoclear.c

```diff
@@ -207,6 +240,7 @@ autoClearWindow (GtkWidget *parent, Account *account)
     // Set the name for this dialog so it can be easily manipulated with css
     gtk_widget_set_name (GTK_WIDGET(data->window), "gnc-id-auto-clear");
 
+    data->ok_button = GTK_WIDGET (gtk_builder_get_object (builder, "ok_button"));
     /* Add amount edit box */
     data->end_value = GNC_AMOUNT_EDIT(gnc_amount_edit_new());
     g_signal_connect(GTK_WIDGET(data->end_value), "activate",
```

Comment:
@Bob-IT quick question -- why does the `end_value` "activate" signal trigger the `gnc_autoclear_window_ok_cb`? Shouldn't it connect to the `ok_button` instead? Is this a bug?

Reply:
@christopherlam It is there to respond to hitting the enter key after inputting amount, otherwise one would enter amount and then need to press OK